Sources/AloeStackView/Protocols/Highlightable.swift

```diff
@@ -13,28 +13,41 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import UIKit
+
 /**
- * Indicates that a row in an `AloeStackView` should change its background color to a
- * highlighted color when a user is touching it.
+ * Indicates that a row in an `AloeStackView` should be highlighted when the user touches it.
  *
  * Rows that are added to an `AloeStackView` can conform to this protocol to have their
- * background color automatically change to a highlighted color when the user is pressing down on
+ * highlightAction(isHighlighting:) will be performed when the user is pressing down on
```

Comment:
Clarity: ... to have their *background color automatically change to a highlighted color (or some other custom behavior defined by the row)* when the user is pressing down on them.